tests/bin/include-submodules/run.t

```diff
@@ -1,45 +1,35 @@
 We need a basic opam project skeleton
 
-    $ cat > CHANGES.md << EOF \
-    > ## 0.1.0\
-    > \
-    > - Some other feature\
-    > \
-    > ## 0.0.0\
-    > \
-    > - Some feature\
-    > EOF
-    $ touch whatever.opam
-    $ cat > dune-project << EOF \
-    > (lang dune 2.4)\
-    > (name whatever)\
-    > EOF
-    $ cat > .gitignore << EOF \
-    > _build\
-    > run.t*\
-    > EOF
+  $ cat > CHANGES.md << EOF
+  > ## 0.1.0
+  > 
+  > - Some other feature
+  > 
+  > ## 0.0.0
+  > 
+  > - Some feature
+  > EOF
+  $ touch whatever.opam
+  $ cat > dune-project << EOF
+  > (lang dune 2.4)
+  > (name whatever)
+  > EOF
+  $ cat > .gitignore << EOF
+  > _build
+  > run.t
+  > EOF
 
 We need to set up a git project for dune-release to work properly
 
-    $ git init 2> /dev/null > /dev/null
-    $ git config user.name "dune-release-test"
-    $ git config user.email "pseudo@pseudo.invalid"
-    $ git add CHANGES.md whatever.opam dune-project .gitignore
-    $ git commit -m "Initial commit" > /dev/null
-    $ dune-release tag -y
-    [-] Extracting tag from first entry in CHANGES.md
-    [-] Using tag "0.1.0"
-    [+] Tagged HEAD with version 0.1.0
+  $ git init > /dev/null 2>&1
+  $ git config user.name "dune-release-test"
+  $ git config user.email "pseudo@pseudo.invalid"
+  $ git add CHANGES.md whatever.opam dune-project .gitignore
+  $ git commit -m "Initial commit" > /dev/null
+  $ dune-release tag -y > /dev/null
 
-Generating the tarball with --include-submodules should call git submodule update --init from within
-the tarball build dir:
+Generating the tarball with `--include-submodules` should call `git submodule
+update --init` from within the tarball build dir:
 
-    $ dune-release distrib --skip-lint --skip-build --skip-test --include-submodule --dry-run
-    ...
-    => chdir _build/whatever-0.1.0.build
-       [in _build/whatever-0.1.0.build]
-    -: exec: git --git-dir .git submodule update --init
-    ...
-    [+] Wrote archive _build/whatever-0.1.0.tbz
-    ...
-    [+] Archive _build/whatever-0.1.0.tbz
+  $ dune-release distrib --skip-lint --skip-build --skip-test --include-submodule --dry-run | grep -- "--init"
```

Comment:
Any particular reason to grep for this rather than `git submodule update`?

Reply:
Yes, with git it is quite easy to add `git [options-here] submodule [maybe-options-here?] update [definintely-options-here]` so I felt that that would've been somewhat brittle, whereas `--init` is unlikely to be removed from the git call. And given the command will be printed it will still check that it is `git submodule update` and not some other `git` subcommand that uses `--init`.

In practice I could also `grep` for `submodule` or `update` but these are more likely to be printed by `dune-release` itself. Which is not a big issue since an additional line isn't a problem, just `--init` seemed to me like the safest bet.

Reply:
Sounds good to me, let's keep `--init`!